--- libnetdude-0.11/src/Makefile.am~	2009-09-15 11:03:52.000000000 +0200
+++ libnetdude-0.11/src/Makefile.am	2009-09-15 11:06:32.000000000 +0200
@@ -32,7 +32,6 @@
 	libnd_raw_protocol.h \
 	libnd_registry.h \
 	libnd_tcpdump.h \
-	libnd_timestamp.h \
 	libnd_tpm.h \
 	libnd_tp.h \
 	libnd_trace.h \
@@ -63,7 +62,6 @@
 	libnd_raw_protocol.c libnd_raw_protocol.h \
 	libnd_registry.c libnd_registry.h \
 	libnd_tcpdump.c libnd_tcpdump.h \
-	libnd_timestamp.h \
 	libnd_tpm.c libnd_tpm.h \
 	libnd_tp.c libnd_tp.h \
 	libnd_trace.c libnd_trace.h \
